### DISCUSSION TITLE
Has the way `invalidate` works changed?

### DISCUSSION THREAD

**Discussion by DePasqualeOrg** [sveltejs/kit#6343] [Q&A] [answered]
Up until one of the recent SvelteKit releases, I could call `invalidate('/some/page')` to reload the props for a given page. After updating to a recent version of SvelteKit, I noticed that I have to call `invalidate()` to get the page's props, and those props aren't reloaded when I specify the path. I've noticed this change across all the pages on which I was previously specifying a path in `invalidate`. Is this change intended, a bug, or something I'm not fully understanding?

**Comment by teemingc** [comment, marked as answer]
Yes, they have changed how invalidate works. It used to require a URL, but now works without a URL too.
Explanations here: https://kit.svelte.dev/docs/load#invalidation

**Reply by DePasqualeOrg** [reply]
Okay, thanks. I definitely missed this change and only noticed it when my app's functionality broke in several places. I just searched the release notes going back more than a month and don't see any mention of this.

**Reply by teemingc** [reply]
Yeah, there weren't any explicit release notes. It was part of the routing system changes
